plugins/package-managers/python/src/test/kotlin/PoetryTest.kt

```diff
@@ -25,17 +25,19 @@ import io.kotest.matchers.file.aFile
 import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNotBe
 
+import org.ossreviewtoolkit.plugins.packagemanagers.python.Pip.Companion.PYPROJECT_FILENAME
+
 class PoetryTest : WordSpec({
     "parseScopeNamesFromPyProject()" should {
         "return the 'main' scope even for a non-existing file" {
-            val pyprojectFile = tempdir().resolve(Pip.PYPROJECT_FILENAME)
+            val pyprojectFile = tempdir().resolve(PYPROJECT_FILENAME)
```

Comment:
I'm curious (really just trying to understand the rationale, no criticism): Does this make the (overall) code more compact in your eyes? Because comparing the number of characters, you're removing 12 chars in favor of adding ~90 chars. Granted, the chars are in a single import that's most of the time collapsed by the IDE, but I'm still interested in the "metric" you follow.

Or is this maybe rather about getting rid of the `Pip.` qualifier, so the refactoring in the next commit touches fewer places? I'd understand that, but it'd be a different rationale IMO.

Reply:
It's about readability. As the reader is spending more time looking at the function implementation vs. looking at the imports, this IMO increases readability of the functions while sacrificing a bit on the readability of the imports. (I didn't aim at reducing the byte size of the file)